blogs/big_data/data_visualization/blog.md

```diff
@@ -0,0 +1,204 @@
+
+# How to create a Tableau dashboard:
+
+This blog will teach you how to create a Tableau dashboard in the following steps:
+* Step 1: Connect to data source
+* Step 2: Create a table with filters
+* Step 3: Create a line graph with a date filter
+* Step 4: Format the dashboard
+
+Step 1: Connect to data source
+- 
+Once you open Tableau, you need to connect to a data source. 
+We will use a Microsoft Excel file for this dashboard.
+
+![](https://i.imgur.com/UUjUMPf.jpg)
+
+
+After connection, raw data:
+
+![](https://i.imgur.com/5o0HVof.jpg)
+
+
+
+Step 2: Create a table with filters
+- 
+In the first sheet, we will create a COVID-19 stat table with state and county filters.
+This step is broken into three parts:
+* Create table
+* Remove abc column
+* Add filters
+
+**Create table**
+This is what you will see on the left side.
+Dimensions display Date, State, and County and Measures display numeric columns, such as Cases and Deaths.
+
+![](https://i.imgur.com/0GSeyCd.jpg)
+
+In the table, we would like to display Date, State, County, Cases, and Deaths columns. So we will drag those column names into "Rows" section.
+
+![](https://i.imgur.com/tLH334O.jpg)
+
+But the table looks weird.
+We need to set Cases, Deaths columns to "Discrete."
+
+![](https://i.imgur.com/YIseNHc.jpg)
+
+Once we change Cases and Deaths to Discrete, this is what it looks like!
+
+This is what we want, except that there is an Abc column.
+![](https://i.imgur.com/d3lQDWo.jpg)
+
+
+**Remove Abc column**
```

Comment:
I like this section for the blog, but I figure this will need to be cut or only linked to in the webinar-version if we want to save on time

Reply:
Yes, I agree that removing ABC column isn't really related to the data visualization. But, you will encounter this issue every time you create a table in Tableau. So, I just it's good to be covered in the blog as well.